Server/src/main/java/Database/TheAuPair/Services/hoursLoggedService.java

```diff
@@ -64,6 +64,9 @@ public int getDateMinutes(String id, String date)
     for (hoursLogged hourLog : hl)
     {
       String [] ts = hourLog.getTimeStart().split(":");
+      if (hourLog.getTimeEnd() == null || hourLog.getTimeEnd().equals(""))
+        continue;
+
       String [] te = hourLog.getTimeEnd().split(":");
```

Comment:
variables "ts" and "te" could be a bit more descriptive - just make them timeStart and timeEnd